sp/src/game/server/items.h

```diff
@@ -104,9 +104,8 @@ class CItem : public CBaseAnimating, public CDefaultPlayerPickupVPhysics
 #endif
 
 #ifdef EZ
-	// TODO: Maybe make EZ_VARIANT a shared enum instead of being within CAI_BaseNPC?
-	typedef CAI_BaseNPC::EZ_VARIANT EZ_VARIANT_ITEM;
-	EZ_VARIANT_ITEM m_tEzVariant;
+	EZ_VARIANT	GetEZVariant() { return m_tEzVariant; }
```

Comment:
Is this method redundant with the definition in BaseNPC now?

Reply:
`CBaseEntity` only has virtual functions, not an actual variant variable. It's up to derived classes to implement their own variables or determine E:Z variant on some other basis.